code/mod_table/mod_table.cpp

```diff
@@ -1528,7 +1528,7 @@ void mod_table_reset()
 	Default_fiction_viewer_ui = -1;
 	Enable_external_shaders = false;
 	Enable_external_default_scripts = false;
-	Default_detail_level = 3; // "very high" seems a reasonable default in 2012 -zookeeper
+	Default_detail_level = DefaultDetailLevel::High; // "high" seems a reasonable default in 2012 -zookeeper
```

Comment:
Pretty small but might be good to update this comment to simply `"High" is a reasonable default`

Reply:
I would say the comment as-is provides useful insight to the last time the default was adjusted and seems a net loss to remove that.

Reply:
Ah yeah, I was just mentioning that b/c zookeeper's statement was originally `Very High` and here it looks like it has been pushed down to just `High`?

Reply:
In that case, the issue isn't the comment it's the default value being ::high instead of ::highest

Reply:
Oh gothca, that makes sense, thanks!